### PR TITLE
Added Oncetold to datafile and Oncetold logo to assets folder.

### PR DESCRIPTION
**Pull request by gagglepod**
I added my new Podcasting 2.0 native hosting platform Oncetold (Beta) to the data file. Go podcasting!

### CODE REVIEW

File: server/data/apps.json [hunk 1]
```diff
@@ -3600,5 +3600,70 @@
...
+    ],
+    "appUrl": "https://oncetold.us",
+    "appIconUrl": "oncetold.png",
+    "platforms": [
+      "Web"
+    ],
+    "supportedElements": [
+      {
+        "elementName": "GUID",
+        "elementURL": "https://www.podhome.fm/docs/podcasting-2-0"
```
Comment (Marzal) [inline server/data/apps.json:3617]: Shouldn't the elementURLs point to the info on your platform?

**Review by Marzal — CHANGES_REQUESTED** [review]
Congrats on the platform

elementURL seems like copy-pasted from podhome.fm

File: server/data/apps.json (hunk 1 above)
Comment (gagglepod) [reply server/data/apps.json:3617]: Yup! I need to edit it. Sorry

**Comment by gagglepod** [comment]
Okay! Corrected the elementURL tags. Thank you for pointing it out! 🤠☕

**Pull request merged**